mm2src/coins/utxo/utxo_builder/utxo_coin_builder.rs

```diff
@@ -667,6 +670,40 @@ pub trait UtxoCoinBuilderCommonOps {
 
         (None, None)
     }
+
+    // Calculates the starting block height based on a given date and the current block height.
+    ///
+    /// # Arguments
+    /// * `date`: The date in seconds representing the desired starting date.
+    /// * `current_block_height`: The current block height at the time of calculation.
+    ///
+    fn calculate_starting_height_from_date(&self, date: u64, current_block_height: u64) -> Result<u64, String> {
+        let buffer = self.conf()["avg_blocktime"]
+            .clone()
+            .as_u64()
+            .ok_or_else(|| format!("avg_blocktime not specified in {} coin config", self.ticker()))?;
+        let buffer = buffer * DAY_IN_HOURS;
```

Comment:
isn't buffer supposed to be blocks per day? if `avg_blocktime` is 1 second this will be 24 blocks per day which is not correct.

Reply:
thanks should be `DAY_IN_SECONDSS/AVG_BLOCKTIME`

Reply:
you can rename buffer to `blocks_per_day`, it makes the whole code more readable. Please pay more attention (in general not in this PR only) to variable naming to make the code readable for anyone that doesn't know the functionalities implemented.